designer/client/src/components/Process/types.ts

```diff
@@ -16,20 +16,13 @@ export type ActionName = string;
 
 export type ProcessVersionId = number;
 
-export type BuildInfoType = {
-    buildTime: string;
-    gitCommit: string;
-    name: string;
-    version: string;
-};
-
 export type ProcessActionType = {
     performedAt: Instant;
     user: string;
     actionName: ActionName;
     commentId?: number;
     comment?: string;
-    buildInfo?: BuildInfoType;
+    modelInfo?: any;
```

Comment:
If its not used lets just remove this field. Better than having an unused any type